### DISCUSSION TITLE
[C#] [Specflow] Allure - No test context is Active

### DISCUSSION THREAD

**Discussion by ricalbuquerque** [allure-framework/allure2#2468] [Bug Reports] [answered]
Hello everyone. 
I've been trying to setup Allure reporter on my specflow project but I'm getting a "System.InvalidOperationException : No test context is active." even when using the "Start project" generator in "https://allurereport.org/start/"
The configuration I provided is:
Language: C#
Framework: Specflow
Target Framework: .Net 8
Unit Test Framework: xUnit.net
My current OS: macOS Sonoma 14.4
Installed allure bin: v2.26.0

Is anyone else experiencing this? Is it a known issue with an easy fix?

Thank you in advance!

**Comment by delatrie** [comment, marked as answer]
Hi, @ricalbuquerque !

That is a known issue with compatibility between Allure.SpecFlow and SpecFlow.xUnit. I doubt we will be able to fix it soon. SpecFlow.NUnit and SpecFlow.MSTest are supported, though.

I advise switching to [Reqnroll](https://reqnroll.net/), a reboot of SpecFlow by Gáspár Nagy, the original author of SpecFlow (since SpecFlow is not actively maintained nowadays). The adapter for Reqnroll ([Allure.Reqnroll](https://www.nuget.org/packages/Allure.Reqnroll)) supports all three unit test frameworks just fine.

**Reply by ricalbuquerque** [reply]
Thank you for the quick reply! 😄
